hack/cherry_pick_pull.sh

```diff
@@ -180,7 +180,7 @@ for pull in "${PULLS[@]}"; do
       (git status --porcelain | grep ^U) || echo "!!! None. Did you git am --continue?"
       echo
       echo "+++ Please resolve the conflicts in another window (and remember to 'git add / git am --continue')"
-      read -p "+++ Proceed (anything but 'y' aborts the cherry-pick)? [y/n] " -r
+      read -p "+++ Proceed (anything apart from 'y' aborts the cherry-pick)? [y/n] " -r
```

Comment:
can we do `other than` instead of `apart from`?

Reply:
Hi @dims Thanks for your suggestions. I have addressed the comments PTAL!!